.pre-commit-config.yaml

```diff
@@ -14,11 +14,11 @@ repos:
     rev: 23.1.0
     hooks:
     -   id: black
--   repo: https://github.com/pycqa/flake8
-    rev: '6.0.0'
+-   repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: 'v0.0.257'
     hooks:
-    -   id: flake8
-        args: ["--ignore=E203,F401,W503", --max-line-length=88]
+        - id: ruff
+          args: [--fix, --exit-non-zero-on-fix]
```

Comment:
this makes ruff automatically remove unused imports

Reply:
see the remove imports here https://github.com/EleutherAI/elk/pull/154/commits/f337f745225e93c714b018bf600d18d757c5779b